sdk/src/main/java/io/dapr/actors/client/IActorProxy.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+package io.dapr.actors.client;
+
+import io.dapr.actors.ActorId;
+
+/**
+ *
+ * Provides the interface for implementation of proxy access for actor service.
+ */
+public interface IActorProxy {
```

Comment:
We don't use the I* convention. Call ActorProxy and rename the implementation to ActorProxyImpl.